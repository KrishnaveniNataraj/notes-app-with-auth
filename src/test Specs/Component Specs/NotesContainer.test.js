import NotesContainer from '../../components/NotesContainer';
import React from 'react';
import { shallow, mount } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';

expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));


let wrapper, notes, handleRemoveNote;
beforeEach(() => {    
    notes= [{
        "noteTitle": "note1",
        "noteDescription": "note1 desc",
        "id": 1
      }]
      handleRemoveNote = jest.fn();
    wrapper = shallow(
        <NotesContainer notes={notes} handleRemoveNote={handleRemoveNote} />
    );
});


test('Render NotesContainer component correctly', () => {        
    expect(wrapper).toMatchSnapshot();
});