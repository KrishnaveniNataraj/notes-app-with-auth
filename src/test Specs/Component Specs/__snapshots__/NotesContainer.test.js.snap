// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render NotesContainer component correctly 1`] = `
<WithStyles(Grid)
  container={true}
  spacing={8}
>
  <WithStyles(Grid)
    item={true}
    key="1"
    sm={4}
    xl={3}
    xs={12}
  >
    <WithStyles(Note)
      handleRemoveNote={[MockFunction]}
      note={
        Object {
          "id": 1,
          "noteDescription": "note1 desc",
          "noteTitle": "note1",
        }
      }
    />
  </WithStyles(Grid)>
</WithStyles(Grid)>
`;
